# Non-repatative reads

# Destination for the read or write
uint8 MSP430=0
uint8 IMU=1
uint8 device

# Address of register to access
uint8 addr

# Num of bytes to read
uint8 size

----

# Read data
uint8[] data